README.md

```diff
@@ -180,3 +201,64 @@ produce early errors because of recursive application of the first rule.
 - If Type(_targetName_) is not `String`, then let _targetName_ be the empty string.
 - Let _status_ be ? SetFunctionName(_F_, _targetName_, `"bound"`).
 - Return _F_.
+
+----
+
+### Bound Constructor Exotic Objects
+
+A bound constructor is an exotic object that wraps a function object with a [[Construct]] internal method. A bound constructor is callable (it has a [[Call]] internal method) and constructible (it has a [[Construct]] internal method). Calling a bound constructor generally results in a call of its wrapped function's constructor.
```

Comment:
The *bound constructor objects* described below don't seem to be constructible to me?